kinetic/distribution.yaml

```diff
@@ -10259,6 +10259,16 @@ repositories:
       url: https://github.com/PX4/px4_msgs-release.git
       version: 1.0.0-3
     status: developed
+  py_script-release:
+    release:
+      packages:
+      - humanoid_shp
+      - humanoid_shp_moveit
+      - py_script
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/dippingconda/py_script-release.git
```

Comment:
And then this release repository should also be called something like `humanoid_shp-release.git`.  Also, the source repository name doesn't match up with the release repository name.